### PR TITLE
Add "prepare" npm script to Gatsby Admin"

### PR DESCRIPTION
**Pull request by mxstbr**
To make sure the freshly built version of Admin is published to npm.

### CODE REVIEW

File: packages/gatsby-admin/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "@emotion/styled": "^10.0.27",
     "@typescript-eslint/eslint-plugin": "^2.28.0",
     "@typescript-eslint/parser": "^2.28.0",
+    "cross-env": "^5.2.1",
```
Comment (wardpeet) [inline packages/gatsby-admin/package.json:18]: this is a dev dependency :)

File: packages/gatsby-admin/package.json [hunk 2]
```diff
@@ -33,6 +34,7 @@
   },
   "scripts": {
     "develop": "gatsby develop",
+    "prepublishOnly": "cross-env NODE_ENV=production yarn run build",
     "build": "gatsby build --prefix-paths",
     "watch": "nodemon --watch src --ext js,ts,tsx,json --exec \"yarn run build\""
```
Comment (wardpeet) [inline packages/gatsby-admin/package.json:39]: nodemon is missing from devDeps

File: packages/gatsby-admin/package.json (hunk 1 above)
Comment (mxstbr) [reply packages/gatsby-admin/package.json:18]: https://github.com/gatsbyjs/gatsby/pull/24644

**Comment by mxstbr** [comment]
Merged in #24810!

**Pull request closed without merging**